src/utils/logger/logger.ts

```diff
@@ -6,10 +6,7 @@ import { SeverityLevel } from './severityLevel';
 
 const logger = winston.createLogger({
   defaultMeta: { service: 'Brain-Service' },
-  format: winston.format.combine(
-    winston.format.timestamp({ format: 'YYYY-MM-DD HH:mm:ss' }),
-    winston.format.json(),
-  ),
+  format: winston.format.combine(winston.format.timestamp({ format: 'YYYY-MM-DD HH:mm:ss' }), winston.format.json()),
```

Comment:
TSLint failed with this line (because it too long), I think the previous change was fine.

Reply:
done